versions.props

```diff
@@ -19,7 +19,6 @@ commons-io:commons-io = 2.1
 io.atomix:* = 1.0.0-rc9
 io.atomix.catalyst:* = 1.1.2
 io.atomix.copycat:* = 1.2.0
-io.dropwizard.metrics:metrics-* = 3.1.1
```

Comment:
gotcha. we may need to leave `io.dropwizard.metrics:metrics-* = 3.1.1` in here for use by `atlasdb-timelock-server` and `atlasdb-dropwizard-bundle`